languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -39150,6 +39150,100 @@ though, and has since been slightly modified)-->
+                        </token>
+                    </marker>
+                    <token regexp="yes">[^,‘’–)\[—.!"\/;\]\|:(\…'«»“”-]+</token>
+                </pattern>
+                <message>Une virgule semble être requise dans ce contexte.</message>
+                <suggestion>\2,</suggestion>
+                <example correction="Bonjour,"><marker>Bonjour</marker> je m'appelle Udo.</example>
+                <example>Salut!</example>
+                <example>Bonjour chez vous. (P)</example>
+                <example>Bonjour camarade. (N)</example>
+                <example>Bonjour les amis. (D)</example>
+                <example>Bonjour Jean. (Z)</example>
+                <example>Bonjour et bienvenue. (C coor)</example>
+                <example>Bonjour toi. (R pers obj)</example>
+                <example>Bonjour chers amis. (J)</example>
+                <example>Bonjour toutes et tous. (A)</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <token postag="SENT_START"/>
+                        <token regexp="yes">bonnes?</token>
+                    <marker>
+                        <token regexp="yes">nuits?|journées?|vacances|après-midi|matinées?
+                            <exception postag="P.*" postag_regexp="yes" scope="next"/>
+                            <exception postag="N.*" postag_regexp="yes" scope="next"/>
+                            <exception postag="D.*" postag_regexp="yes" scope="next"/>
+                            <exception postag="Z.*" postag_regexp="yes" scope="next"/>
+                            <exception postag="C coor.*" postag_regexp="yes" scope="next"/>
+                            <exception postag="R pers obj.*" postag_regexp="yes" scope="next"/>
+                            <exception postag="J.*" postag_regexp="yes" scope="next"/>
+                            <exception postag="A.*" postag_regexp="yes" scope="next"/>
+                        </token>
+                    </marker>
+                    <token regexp="yes">[^,‘’–)\[—.!"\/;\]\|:(\…'«»“”-]+</token>
+                </pattern>
+                <message>Une virgule semble être requise dans ce contexte.</message>
+                <suggestion>\3,</suggestion>
+                <example correction="après-midi,">Bonne <marker>après-midi</marker> je rentre chez moi.</example>
+                <example>Bonnes vacances!</example>
+                <example>Bonne nuit chez vous. (P)</example>
+                <example>Bonne nuit camarade. (N)</example>
+                <example>Bonne nuit les amis. (D)</example>
+                <example>Bonne nuit Jean. (Z)</example>
+                <example>Bonne nuit et bienvenue. (C coor)</example>
+                <example>Bonne nuit toi. (R pers obj)</example>
+                <example>Bonne nuit chers amis. (J)</example>
+                <example>Bonne nuit toutes et tous. (A)</example>
```

Comment:
This kind of technical remark cannot be included in an example (examples will be visible to users at some point, so we cannot put any technical stuff in there). If you want to include a comment, you can do that in a `<!-- … -->` after the example.